src/app/public/modules/popover/popover.directive.ts

```diff
@@ -48,18 +48,35 @@ import {
 })
 export class SkyPopoverDirective implements OnChanges, OnDestroy {
 
+  /**
+   * References the popover component to display. Add this directive to the trigger element that opens the popover.
+   */
   @Input()
   public skyPopover: SkyPopoverComponent;
 
+  /**
+   * Specifies the horizontal alignment of the popover in relation to the trigger element.
+   * @default "center"
+   */
   @Input()
   public skyPopoverAlignment: SkyPopoverAlignment;
 
+  /**
+   * Specifies the placement of the popover in relation to the trigger element.
+   * @default "above"
+   */
   @Input()
   public skyPopoverMessageStream = new Subject<SkyPopoverMessage>();
 
+  /**
+   * Specifies the user action that displays the popover.
+   */
   @Input()
   public skyPopoverPlacement: SkyPopoverPlacement;
 
+  /**
+   * Provides an observable to send commands to the popover that respect the `SkyPopoverMessage` type.
+   */
   @Input()
   public skyPopoverTrigger: SkyPopoverTrigger = 'click';
```

Comment:
I noticed `SkyPopoverAlignment` and `SkyPopoverPlacement` have descriptions now in the docs. However, the type for `SkyPopoverTrigger` does not. Should it?

Reply:
Good catch.